streams/writable-streams/aborting.js

```diff
@@ -1119,7 +1118,14 @@ promise_test(t => {
 
 promise_test(t => {
   const events = [];
-  const ws = recordingWritableStream();
+  let startResolve;
```

Comment:
It looks like this is a different test. The old one was correct; can you add a new one that tests the new code path you're working through, instead of modifying the existing one?